neo4j-app/neo4j_app/core/config.py

```diff
@@ -41,6 +41,8 @@ class AppConfig(LowerCamelCaseModel, IgnoreExtraModel):
     es_keep_alive: str = "1m"
     neo4j_app_host: str = "127.0.0.1"
     neo4j_app_log_level: str = "INFO"
+    neo4j_app_migration_timeout_s: float = 60 * 5
+    neo4j_app_migration_wait_s: float = 1
```

Comment:
Yes you're right, it's a throttle, if the DB is locked because of a migration currently running then the current thread goes to sleep for `neo4j_app_migration_throttle_s` and tries to acquire the lock after that until it gets it or until all migrations are complete.